Update RepoInfo.json

Sampletext282 opened the pull request.

apple1417 commented:
Looks like you've currently got a trailing comma in the versions dict. Python uses strictly pure json, where that's not allowed, hence causing the errors. Let me know once you've fixed it and I'll rerun

Sampletext282 commented:
sorry my bad. fixed it

> Looks like you've currently got a trailing comma in the versions dict. Python uses strictly pure json, where that's not allowed, hence causing the errors. Let me know once you've fixed it and I'll rerun

Sorry, my bad. Fixed it, thank you
